src/spaced/spaced/scenes/game.cpp

```diff
@@ -48,16 +49,19 @@ void Game::on_loaded() {
 	m_player.foam_particles.set_texture(foam_texture);
 
 	auto const& rgbas = get_services().get<Styles>().rgbas;
-	auto spawn = [this] {
+	auto spawn = [this, rgbas] {
 		auto ret = std::make_unique<Creep>(get_services(), this);
-		ret->shape.tint = bave::yellow_v;
+		ret->shape.tint = random_in_range(-1.f, 1.f) < 0.f ? rgbas["orange"] : rgbas["milk"];
```

Comment:
Why not `random_in_range(0, 1) == 0`? Floating point randomness is more computationally expensive, and you just want a binary value, not a theoretically infinite range of floats.